include/inexor/vulkan-renderer/wrapper/window.hpp

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include <GLFW/glfw3.h>
+#include <spdlog/spdlog.h>
+#include <vulkan/vulkan.h>
+
+#include <cassert>
```

Comment:
`cassert` can be moved to the source file.

Reply:
Done.